.github/workflows/docker.yml

```diff
@@ -0,0 +1,21 @@
+name: Docker Compose Actions Workflow
+on: [push, pull_request]
```

Comment:
Missed this, sorry -- do we want to restrict at least push to `master`?

Reply:
"push" is the only way to test workflows afaict, I don't think we want to restrict to master